### DISCUSSION TITLE
(ERROR(820): VENT

### DISCUSSION THREAD

**Discussion by AbdelMoumene01** [firemodels/fds#12860] [General]
’m encountering an error. Could the issue be related to using the MULT function? 
&HEAD CHID='DRRC_1RM', TITLE='DRRC_1RM'/
/MESH obttained with DX = 1 meter/
&MESH IJK=140,165,40, XB=0,140,0,165,0,40/
&TIME T_END=10/
&MISC RADIATION=.TRUE.,TMPA=43/
&MISC HUMIDITY=18/
&SURF ID='HOT' ,TMP_FRONT = 43/

&OBST XB= 3,118,3,3,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/                /cuvette de rétention GRAY /
&OBST XB= 3,3,3,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/                 /cuvette de rétention GRAY /
&OBST XB= 118,118,3,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/             /cuvette de rétention GRAY /
&OBST XB= 3,118,68,68,0,3, COLOR='DIM GRAY',SURF_ID='HOT'/              /cuvette de rétention GRAY /
&OBST XB= 28,28,3,68,0,1.5, COLOR='DIM GRAY',SURF_ID='HOT'/             /cuvette de rétention GRAY /
&OBST XB= 28,118,38,38,0,1.5, COLOR='DIM GRAY',SURF_ID='HOT'/           /cuvette de rétention GRAY /

&SURF ID='TK_Gasoline_01', COLOR='SKY BLUE'/
&MULT ID='cube array_1', DX=1,DY=1,DZ=1, I_UPPER=16,J_UPPER=16,K_UPPER=8/
&OBST XB= 8,9,12,13,0,8,SURF_ID='HOT', 
      MULT_ID='cube array_1'
      SURF_ID='TK_Gasoline_01'
      SHAPE='CYLINDER'
      RADIUS=8
      HEIGHT=8    
      XYZ=16,20,0
      ORIENTATION=0,0,1/                                                    /bac essence sans plamb 01 /

&SURF ID='TK_Gasoline_02', COLOR='SKY BLUE'/
&MULT ID='cube array_2', DX=1,DY=1,DZ=1, I_UPPER=16,J_UPPER=16,K_UPPER=8/
&OBST XB= 8,9,46,47,0,8, SURF_ID='HOT',
      MULT_ID='cube array_2'
      SURF_ID='TK_Gasoline_02'
      SHAPE='CYLINDER'
      RADIUS=8
      HEIGHT=8    
      XYZ=16,54,0
      ORIENTATION=0,0,1/                                                     /bac essence sans plamb 02 /
&SURF ID='BURNER',COLOR='RED', HRRPUA=200 /
&REAC FUEL='Gasoline', C=8, H=18, O=0, N=0, CO_YIELD=0.01, SOOT_YIELD=0.01 /
&VENT XB=8,9,12,13,8,8, XYZ=16,20,8, RADIUS=8, COLOR='RED', SURF_ID='BURNER'/

 i have this error
i try to use IOR=3, but nothing
(ERROR(820): VENT 1 has no solid backing or an orientation index (IOR) is needed)

**Comment by Er9y714** [comment]
The vent seems to be inserted into a solid object. There is not any side that is in contact with gas. 

![image](https://github.com/firemodels/fds/assets/26166281/b60a9128-9109-4960-8ee3-d7836d293a31)

**Reply by AbdelMoumene01** [reply]
Of course I understand, but I'm wondering why I can't create a surface on top of the cylinder using the "MULT" function. Could it be that the "MULT" function doesn't produce a solid surface like the obstruction?

**Reply by drjfloyd** [reply]
If only there were some document, like a User's Guide that had say a section on the basics of defining obstructions that discussed the various options for applying surfaces to obstructions.

**Reply by AbdelMoumene01** [reply]
I didn't find in the user's guide how to add a circular burning surface on top of a cylinder.

**Reply by drjfloyd** [reply]
You are using OBST and MULT for your inputs.  All MULT does, is make copies of the OBST per the instructions on the multi line. If you were to look open yur smv file in smokeview and show the obstacle outlines, you would see your cylinder is composed of multiple copies of your OBST input. Whatever boundary conditions you have specified for that OBST will apply to every copy of the OBST made by MULT.  So if you specifed your OBST input such that the top surface of the OBST was your burning surface and the sides and bottom of the OBST were the tank, then every OBST copy made by MULT would have its top surface be your burning surface and the sides and bottom be the tank. There are three ways you can specify surfaces for OBST. SURF_ID is one of the them. There are two others, and both allow you to specify different SURF for difference faces of an OBST. If you look at Table 23.20 you can find every input that is possible for OBST in alphabetical order with hyperlinks to where they are discussed in the Guide.

**Reply by AbdelMoumene01** [reply]
I thank you very much, sir, for finding the section discussing this matter. However, I noticed a small error. The radiative or thermal flux, when I retrieved the results from the different captures, does not start at zero, even though I did not specify an initial condition for the flux. So, I wonder what I should check here?

&TIME T_END=300/
&MISC RADIATION=.TRUE./
&INIT XB=0,180,45,220,0,40, TEMPERATURE=43. /
&MISC HUMIDITY=18/
&SURF ID='HOT' ,TMP_FRONT=43/

**Comment by drjfloyd** [comment]
Why would it be zero?  Your ambient temperature (TMPA) is 20 C.  You initialize some volume to 43 C. Your surface starts at 20 C with a 1 second TANH ramp to 43 C.  You do not have isothermal conditions at the start. Please take the time to study the guides to learn how FDS works, what assumptions and defaults it uses, etc.

**Reply by AbdelMoumene01** [reply]
Okay, I will try to read all sections. Thank you very much, sir.
